src/main/java/command/ExitCommand.java

```diff
@@ -0,0 +1,44 @@
+package command;
+import task.*;
+import duke.*;
+
+public class ExitCommand extends Command {
+
+    /**
+     * Saves taskList to file, then calls showResponse with exit message.
+     *
+     * @param tasks Current TaskList.
+     * @param ui Ui object.
+     * @param storage Storage object.
+     */
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
+        // perform saving of taskList to disk here
+        storage.save(tasks);
+
+        String response = respond();
+        String result = ui.showResponse(response);
+
+        return result;
+    }
+
+    /**
+     * Returns true to indicate ExitCommand exits the program.
+     *
+     * @return True.
+     */
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+
+    /**
+     * Returns exit message.
+     *
+     * @return Exit message.
+     */
+    public String respond() {
+        String response = "Bye. Hope to see you again soon!";
```

Comment:
Would it better to make this string response into a [named variable](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)? This would make the code easier to understand too.